Source/WebKit/Shared/EditorState.h

```diff
@@ -48,24 +48,23 @@ class TextStream;
 
 namespace WebKit {
 
-enum TypingAttributes {
-    AttributeNone = 0,
-    AttributeBold = 1,
-    AttributeItalics = 2,
-    AttributeUnderline = 4,
-    AttributeStrikeThrough = 8
+enum class TypingAttribute : uint8_t {
+    Bold          = 1 << 0,
+    Italics       = 1 << 1,
+    Underline     = 1 << 2,
+    StrikeThrough = 1 << 3,
 };
 
-enum TextAlignment {
-    NoAlignment = 0,
-    LeftAlignment = 1,
-    RightAlignment = 2,
-    CenterAlignment = 3,
-    JustifiedAlignment = 4,
+enum class TextAlignment : uint8_t {
+    NoAlignment,
```

Comment:
May be nicer to call this `None` now that we have to specify the enum name (and the name already says "Alignment").

Reply:
What even does TextAlignment::None mean? How can you have text with no alignment!

Reply:
it means `NSTextAlignmentNatural` apparently!

Reply:
s maybe rename from NoAlignment to Natural?